includes/Hooks.php

```diff
@@ -352,7 +395,44 @@ public function onLocalUserCreated( $user, $autocreated ) {
 		// New users get echo preferences set that are not the default settings for existing users.
 		// Specifically, new users are opted into email notifications for thanks.
 		if ( !$user->isTemp() && !$autocreated ) {
-			$this->userOptionsManager->setOption( $user, 'echo-subscriptions-email-edit-thank', true );
+			$userOptionsManager = MediaWikiServices::getInstance()->getUserOptionsManager();
+			$userOptionsManager->setOption( $user, 'echo-subscriptions-email-edit-thank', true );
+		}
```

Comment:
UserOptionsManager is now a dependency in this class
```suggestion
			$this->userOptionsManager->setOption( $user, 'echo-subscriptions-email-edit-thank', true );
		}
```